packages/sommelier.rb

```diff
@@ -3,112 +3,172 @@
 class Sommelier < Package
   description 'Sommelier works by redirecting X11 programs to the built-in ChromeOS Exo Wayland server.'
   homepage 'https://chromium.googlesource.com/chromiumos/platform2/+/HEAD/vm_tools/sommelier/'
-  version '20210109-5'
+  version '20211110'
   license 'BSD-Google'
-  compatibility 'all'
+  compatibility 'aarch64, armv7l, x86_64'
```

Comment:
Is i686 a lost cause?

Reply:
Did sommelier actually ever work on i686? Is there a working wayland compositor on there? Or do those devices just use X11?

Reply:
AFAIK Chromium OS 58 is still using `x11+tty`, I don't think we need `sommelier` on it... (however, some patches is needed to run regular x11 apps in x11 chrome os wm)

Reply:
What patches are needed for x11 apps on `x11+tty` on the `i686` machines?